### PR TITLE
KVM 2021 Talk

### PR DESCRIPTION
**Pull request by gqlo**
KVM 2021 Talk

### CODE REVIEW

File: content/ja/post/li-kvm-forum-21/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Mr. Guoqing Li and Dr.Dario Faggioli's Collabrative Talk at KVM FORUM 2021"
```
Comment (keichi) [inline content/ja/post/li-kvm-forum-21/index.md:4]: Need a space after Dr.

File: content/ja/post/li-kvm-forum-21/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+subtitle: ""
+summary: ""
+authors: ["Guoqing Li"]
+tags: ["Cloud Computing", "Virtualization Performance", "KubeVirt", "Kubernetes"]
```
Comment (keichi) [inline content/ja/post/li-kvm-forum-21/index.md:8]: Please change `"Cloud Computing"` to `"Cloud"`. This way the post will be linked from this page:
https://sdlab.naist.jp/tags/cloud/

File: content/ja/post/li-kvm-forum-21/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Projects (optional).
+#   Associate this post with one or more of your projects.
+#   Simply enter your project's folder or file name without extension.
+#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
+#   Otherwise, set `projects = []`.
+projects: []
+---
+
+Mr.Guoqing Li, a master student from SDLab and Dr.Dario Faggioli, a virtualization specialist from SUSE lab gave a joint [talk](https://youtu.be/x_czS9Iuo2o) on the topic of KubeVirt: The cost of Containerizing VMs at [KVM FORUM 2021](https://events.linuxfoundation.org/kvm-forum/).
```
Comment (keichi) [inline content/ja/post/li-kvm-forum-21/index.md:31]: Need a space after Mr. and Dr.

File: content/ja/post/li-kvm-forum-21/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+KubeVirt is an add-on for Kubernetes to manage both containers and VMs in a unified manner. They evaluated the performance impact of resource usage accounting and limitation caused by container runtimes, as well as the achitectural limitation resulted from integrating QEMU/KVM into Kubernetes stack.
+
+They showed that Kubevirt's limitation on configuring NUMA CPU topology can slow down the CPU performance up to 60%, and proper tunning can mitigate this CPU performance overhead to some degree but further improvement is needed since this will not only have the impact on theperformance, but can also lead to issues with live migrations.
+
+The introduction and presentation slides of the talk is available at [here](https://events.linuxfoundation.org/kvm-forum/program/schedule/).
```
Comment (keichi) [inline content/ja/post/li-kvm-forum-21/index.md:37]: https://sched.co/ke2r would be better as it is a direct link to your talk.

**Review by keichi — CHANGES_REQUESTED** [review]
Please create your post under `content/en` instead of `content/ja`. I will add a Japanese translation afterward.

**Comment by keichi** [comment]
Hugo build is failing -- please move your post under `content/en/post/` and add a _relative_ symlink.

**Comment by gqlo** [comment]
> Hugo build is failing -- please move your post under `content/en/post/` and add a _relative_ symlink.

Interesting. i had to copy the folders to another location, then I can create a relative symlink, otherwise it's created as a file not a folder.

**Comment by keichi** [comment]
FYI this is the correct way to do it:
```
cd content/en/post
ln -s ../../ja/post/li-kvm-forum-21 li-kvm-forum-21
```

**Review by keichi — APPROVED** [review]

**Pull request merged**